ROMFS/px4fmu_common/init.d/4900_crazyflie

```diff
@@ -61,7 +61,7 @@ then
 	param set PWM_MAX 255
 
 	# Run the motors at 328.125 kHz (recommended)
-	param set PWM_RATE 3921
+	param set PWM_RATE 400
```

Comment:
Reading on some crazyflie forums I read that it is recommended to use much higher rates. What is the reason for this change?